simulatorApp/src/Makefile

```diff
@@ -15,6 +15,14 @@ simulator_SRCS += simulator_registerRecordDeviceDriver.cpp
 simulator_SRCS_DEFAULT += simulatorMain.cpp
 simulator_SRCS_vxWorks += -nil-
 
+# Link QSRV (pvAccess Server) if available
+ifdef EPICS_QSRV_MAJOR_VERSION
+    simulator_LIBS += qsrv
+    simulator_LIBS += $(EPICS_BASE_PVA_CORE_LIBS)
+    simulator_DBD += PVAServerRegister.dbd
+    simulator_DBD += qsrv.dbd
+endif
+
```

Comment:
This seems to be already covered by the `WITH_QSRV` and `WITH_PVA` macros in [commonDriverMakefile](https://github.com/areaDetector/ADCore/blob/master/ADApp/commonDriverMakefile#L21-L38), which are already properly defined by [epics-in-docker](https://github.com/cnpem/epics-in-docker/blob/v0.12.0/base/install_area_detector.sh#L48-L49). Have you tested it and it didn't work out?

Reply:
Hmm, I tried `pvget` with PVs from them and it didn't work. Will investigate further.

Reply:
Indeed, it still worked without this commit. Must have been some transient error or typo on my side.

Thanks!